src/OSPSuite.FuncParser/OSPSuite.FuncParser.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>net472;netstandard2.0</TargetFrameworks>
```

Comment:
We need this multi-target so that when appropriate, we can use the `dllimport` name convention.

That means no more net472 in Linux, but I think that's not a problem. Any other configuration should work by specifying the appropriate top-level projects target framework.